### PR TITLE
Add Secure Defaults: Cloud Native 8

### LINKED ISSUE #734: Guideline for Applying Secure Defaults
Description: As part of #480 , a discussion sparked about what does _secure defaults_ and applying it to a project mean for us and for the community in general. This issue is an attempt to reasonably address that through community discussions 

Impact: Identify state of the art and carve content to articulate this in a standalone doc, which could be adopted in other Security TAG deliverables

Some questions we can attempt to address:

- How does secure defaults, impact new user experience, stability, backwards compatibility?
- Is there a golden middle ground somewhere that can be articulated?
- Which projects have done this well in the past?
- Does transparent security mean the same as secure defaults?
- Are there any case studies from end users (security conscious ones especially) that we could benefit from?
- Is this even something that is theoretical possible but never realistically achievable?

TO DO
- [x] Security TAG Leadership Representative: @TheFoxAtWork 
- [x] Project leader(s): @PushkarJ 
- [x] Project Members: Authors and contributors to the guideline
- [x] Scope: Create a high level guideline to successfully enable a project to adopt secure defaults. Each guideline will be brief, with rationale, "how-to" and example
- [x] Deliverable(s): https://docs.google.com/document/d/1Y1OCgto48Woc0UsZHq7zHuYOFGYI_DTpC2o7k7KYeaA/edit# 
- [ ] Due date: 31 October 2021
- [x] Slack Channel (as needed): #tag-security 
- [x] Meeting Time & Day: Regular meetings
- [ ] Next Step: Decide where and how to publish and adop this guidance

### PR DESCRIPTION
**Pull request by PushkarJ**
Fixes #734 

Related to #832 

Only spelling failures are for people names.

### CODE REVIEW

File: security-whitepaper/secure-defaults-cloud-native-8.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# **Secure Defaults: Cloud Native 8**
```
Comment (lumjjb) [inline security-whitepaper/secure-defaults-cloud-native-8.md:1]: nits: the headers don't need the bold, it should be there by default

**Review by lumjjb — APPROVED** [review]
minor nit else LGTM!

File: security-whitepaper/secure-defaults-cloud-native-8.md (hunk 1 above)
Comment (PushkarJ) [reply security-whitepaper/secure-defaults-cloud-native-8.md]: Good call out. The docs to markdown plugin added it and I missed it! Fixed it now.

**Comment by TheFoxAtWork** [comment]
Still a spelling issue but otherwise LGTM

File: security-whitepaper/secure-defaults-cloud-native-8.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
+# Secure Defaults: Cloud Native 8
+
+**Author**: Pushkar Joglekar
+
+**Contributors** (alphabetical order): Andres Vega, Emily Fox, Faraz Angabini,
+Robert Clark, Robert Van Voorhees
```
Comment (lumjjb) [inline security-whitepaper/secure-defaults-cloud-native-8.md:6]:
```suggestion
<!-- cspell:disable -->
**Author**: Pushkar Joglekar

**Contributors** (alphabetical order): Andres Vega, Emily Fox, Faraz Angabini,
Robert Clark, Robert Van Voorhees
<!-- cspell:enable -->
```

**Comment by PushkarJ** [comment]
Thanks @lumjjb for adding `cspell` tags on the author names. Will merge it since all the checks including spellings are passing.

**Pull request merged**